frontend/src/components/cars-categories/cars-categories.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import { mockBrands } from '@components/cars-categories/mockBrands';
+
+import { Brand } from '../../types/brand.type';
+import styles from './styles.module.scss';
+
+const CarsCategories: FC = () => {
+  const [brands, setBrands] = useState<Brand[]>([]);
+  const [activeId, setActiveId] = useState<number>();
+
+  useEffect(() => {
+    if (mockBrands.length) {
+      setBrands(mockBrands);
+    }
+  }, []);
+
+  return (
+    <div className={styles.container}>
+      {brands.map((brand) => (
+        <div
+          key={brand.marka_id}
+          className={
+            brand.marka_id === activeId
+              ? styles.linkWrapper + ' ' + styles.active
+              : styles.linkWrapper
```

Comment:
I don't think that we need to store the active link id state. Because after the click, we would be redirected to another page, and that state change won't be even seen.

When I said "Desing the active state", I meant the [CSS's `:active`](https://developer.mozilla.org/en-US/docs/Web/CSS/:active) :slightly_smiling_face: 
So you just need to have not the `styles.active`, but `.linkWrapper:active` (or any other name of the class which will show the brand tile)